### PR TITLE
Level2->Boss->Menus

### PR DESCRIPTION
**Pull request by miquelmiro3**
Finishing level2 goes to boss killing boss (at the exact moment) goes to menus.

### CODE REVIEW

**Review by monikamnc — APPROVED** [review]
Txatxi pistatxi

File: Gameplay/entities/enemies/BossController.cpp [hunk 1]
```diff
@@ -157,6 +157,7 @@ void Hachiko::Scripting::BossController::CombatController()
 
 	if (combat_stats->_current_hp <= 0)
 	{
+		level_manager->GoalReached();
```
Comment (Bernatmago) [inline Gameplay/entities/enemies/BossController.cpp:160]: I would do this inside the dead state after a timer or after a button is pressed but for now its ok like this

**Review by Bernatmago — APPROVED** [review]
Arigatito important stuff

File: Gameplay/entities/enemies/BossController.cpp (hunk 1 above)
Comment (miquelmiro3) [reply Gameplay/entities/enemies/BossController.cpp:160]: Yep but for now its okey + i dont want to block the scene

**Pull request merged**